docs/monitoring-aws-lambda.asciidoc

```diff
@@ -28,6 +28,30 @@ image:images/architecture-white.png[image showing data flow from lambda function
 
 By using an AWS Lambda extension, Elastic APM agents can send data to a local Lambda extension process, and that process will forward data on to APM Server asynchronously. The Lambda extension ensures that any potential latency between the Lambda function and the APM Server instance will not cause latency in the request flow of the Lambda function itself.
 
+[[aws-lambda-overhead]]
+== Performance impact and overhead
+As described in the section on <<aws-lambda-arch>>, using Elastic APM with AWS Lambda requires to include the Elastic APM AWS Lambda extension and a corresponding Elastic APM agent to the Lambda runtime.
```

Comment:
```suggestion
As described in the section on <<aws-lambda-arch>>, using Elastic APM with AWS Lambda requires one to include the Elastic APM AWS Lambda extension and a corresponding Elastic APM agent with the Lambda function.
```

Grammar. Totally feel free to ignore these suggestions. :)

Reply:
I'll add to Trent's suggestions with this:

```suggestion
As described in <<aws-lambda-arch>>, using Elastic APM with AWS Lambda requires adding both the Elastic APM AWS Lambda extension and a corresponding Elastic APM agent to the Lambda runtime.
```